<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: #FFDEBA;
	padding: 6px;
}

QPushButton::hover {
	background-color: orange;
	color: white;
}

*[ansState=&quot;true&quot;] {
	background-color: green;
}

*[ansState=&quot;false&quot;] {
	background-color: red;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,4,0">
   <property name="bottomMargin">
    <number>42</number>
   </property>
   <item>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: #FFDEBA;
	font: bold 14px;
	padding: 6px;
}

*[ansState=&quot;true&quot;] {
	background-color: green;
}

*[ansState=&quot;false&quot;] {
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>The Game</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>42</number>
       </property>
       <property name="leftMargin">
        <number>48</number>
       </property>
       <property name="rightMargin">
        <number>48</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="guessKeywordBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover {
	background-color: orange;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Guess keyword</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="guiessMeaningBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover {
	background-color: orange;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Guess meaning</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="questionLabel">
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="btnLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="1" column="1">
          <widget class="TextWrapButton" name="ans4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="TextWrapButton" name="ans1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 2px;
border-radius: 10px;
border-color: #FFDEBA;
font: bold 14px;
min-width: 10em;
 padding: 6px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="TextWrapButton" name="ans2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="ansState" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="TextWrapButton" name="ans3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextWrapButton</class>
   <extends>QPushButton</extends>
   <header>textwrapbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
